README.md

```diff
@@ -1,3 +1,7 @@
+> # ⛔️ DEPRECATED: [js-ipfs-http-client-lite](https://github.com/ipfs-shipyard/js-ipfs-http-client-lite) has been superseded by [js-kubo-rpc-client](https://github.com/ipfs/js-kubo-rpc-client)
```

Comment:
```suggestion
> # ⛔️ DEPRECATED: [js-ipfs-http-client-lite](https://github.com/ipfs-shipyard/js-ipfs-http-client-lite) has been superseded by [kubo-rpc-client](https://github.com/ipfs/js-kubo-rpc-client)
```

Personal preference, but I prefer using the npmjs module name when using link labels.